feel-engine/README.md

```diff
@@ -67,6 +67,14 @@ Or using Java:
 }
 ```
 
+Enable FEEL External-Functions in the configuration: (not recommended)
+
+```
+new FeelEngine(configuration = Configuration(externalFunctionsEnabled = true))
+// or
+new FeelEngine.Builder().enableExternalFunctions(true).build()
+```
+
```

Comment:
```suggestion
Enabling external functions is not recommended because external functions make it possible to call arbitrary Java methods. This poses a potential security risk. 

Enabling this feature can be safe if you have full control of the expressions that will be evaluated (i.e. none of them are entered by a user or generated by an external system). In all other cases this feature is considered unsafe.

An alternative to external functions is the use of (Function Providers)[https://camunda.github.io/feel-scala/function-provider-spi]
```